--- origsrc/kgraphviewer-2.1.1/src/part/DotGraphParsingHelper.cpp	2010-11-12 02:07:15.000000000 -0600
+++ src/kgraphviewer-2.1.1/src/part/DotGraphParsingHelper.cpp	2013-01-22 23:36:13.600237400 -0600
@@ -26,10 +26,7 @@
 #include "graphedge.h"
 
 #include <boost/throw_exception.hpp>
-#include <boost/spirit/core.hpp>
-#include <boost/spirit/utility/distinct.hpp>
-#include <boost/spirit/utility/loops.hpp>
-#include <boost/spirit/utility/confix.hpp>
+#include <boost/spirit/include/classic.hpp>
 
 
 #include <iostream>
