--- origsrc/kgraphviewer-1.0.4/kgraphviewer/src/part/Makefile.am	2007-07-14 07:11:58.000000000 -0500
+++ src/kgraphviewer-1.0.4/kgraphviewer/src/part/Makefile.am	2007-08-29 20:20:26.656250000 -0500
@@ -24,10 +24,10 @@
 FontsCache.cpp \
 kgraphviewer_partsettings.kcfgc
 
-libkgraphviewerpart_la_LIBADD = ../printing/libkgraphviewerprinting.la 
+libkgraphviewerpart_la_LIBADD = ../printing/libkgraphviewerprinting.la $(LIB_KHTML) $(LIB_KPARTS) $(LIB_KIO) $(LIB_KDEUI) $(LIB_KDECORE) $(LIB_QT)
 ## $(top_builddir)/kdeprint/libkdeprint.la $(top_builddir)/kutils/libkutils.la $(top_builddir)/kjs/libkjs.la $(LUA_LIBS)
 
-libkgraphviewerpart_la_LDFLAGS = $(all_libraries) $(LIB_QT) $(LIB_KPARTS) $(LIB_KHTML)
+libkgraphviewerpart_la_LDFLAGS = $(all_libraries) $(KDE_PLUGIN)
 
 
 # These paths are KDE specific. Use them:
--- origsrc/kgraphviewer-1.0.4/kgraphviewer/src/printing/KgvPageLayoutDia.h	2007-07-14 07:11:58.000000000 -0500
+++ src/kgraphviewer-1.0.4/kgraphviewer/src/printing/KgvPageLayoutDia.h	2007-08-29 20:01:05.453125000 -0500
@@ -194,7 +194,7 @@
 
 private slots:
     void sizeUpdated(KgvPageLayout &layout);
-    void columnsUpdated(KgvColumns &columns);
+//    void columnsUpdated(KgvColumns &columns);
 
 private:
     KgvPageLayoutSize *m_pageSizeTab;
--- origsrc/kgraphviewer-1.0.4/kgraphviewer/src/printing/Makefile.am	2007-07-14 07:11:58.000000000 -0500
+++ src/kgraphviewer-1.0.4/kgraphviewer/src/printing/Makefile.am	2007-08-29 00:28:23.968750000 -0500
@@ -23,8 +23,8 @@
 KgvPageLayoutHeader.cpp \
 KgvPageLayoutSize.cpp
 
-libkgraphviewerprinting_la_LDFLAGS = $(all_libraries)
-libkgraphviewerprinting_la_LIBADD = $(LIB_KDECORE) $(LIB_QT) $(LIB_KIO) $(LIB_KPARTS) $(LIB_KHML) $(LIB_KDEPRINT)
+libkgraphviewerprinting_la_LDFLAGS = $(all_libraries) -no-undefined
+libkgraphviewerprinting_la_LIBADD = $(LIB_KHML) $(LIB_KDEPRINT) $(LIB_KPARTS) $(LIB_KIO) $(LIB_KDEUI) $(LIB_KDECORE) $(LIB_QT)
 
 kgraphviewerprintingincludedir = $(includedir) $(includedir)/src $(includedir)/printing
 
--- origsrc/kgraphviewer-1.0.4/kgraphviewer/src/printing/simpleprintingcommand.cpp	2007-07-14 07:11:58.000000000 -0500
+++ src/kgraphviewer-1.0.4/kgraphviewer/src/printing/simpleprintingcommand.cpp	2007-08-29 20:07:23.953125000 -0500
@@ -267,7 +267,7 @@
     m_pageSetupDialog = new QDialog(0,"glurp",false);
     QMap<QString,QString> map;
     map["action"]=="pageSetup";
-    map["title"]==m_graphView->dotFileName();
+//    map["title"]==m_graphView->dotFileName();
     QVBoxLayout *lyr = new QVBoxLayout(m_pageSetupDialog);
     KGVSimplePrintingPageSetup* sppsb = new KGVSimplePrintingPageSetup(this, m_graphView, m_pageSetupDialog, &map);
     if (m_previewWindow != 0)
--- origsrc/kgraphviewer-1.0.4/kgraphviewer/src/printing/simpleprintingpagesetup.cpp	2007-07-14 07:11:58.000000000 -0500
+++ src/kgraphviewer-1.0.4/kgraphviewer/src/printing/simpleprintingpagesetup.cpp	2007-08-29 20:09:02.484375000 -0500
@@ -111,7 +111,7 @@
 	if (m_graphView) {
 		m_origCaptionLabelText = m_contents->captionLabel->text();
     m_contents->headerTitleLineEdit->setText((*args)["title"]);
-		m_contents->captionLabel->setText( m_origCaptionLabelText.arg( KURL::fromPathOrURL(m_graphView->dotFileName()).fileName() ) );
+//		m_contents->captionLabel->setText( m_origCaptionLabelText.arg( KURL::fromPathOrURL(m_graphView->dotFileName()).fileName() ) );
 	}
 	connect(m_contents->headerTitleLineEdit,SIGNAL(textChanged(const QString&)), 
 		this, SLOT(slotTitleTextChanged(const QString&)));
